examples/simple/igraph_get_laplacian_sparse.c

```diff
@@ -127,6 +127,7 @@ int test_normalized_laplacian(const igraph_vector_t *w, igraph_bool_t dir) {
     igraph_vector_int_t vec;
     igraph_vector_t *weights = 0;
     igraph_bool_t ok = 1;
+
```

Comment:
Please revert changes that to files that are not related to this PR.

Reply:
I don't recall adding a blank line here. I think I did this accidentally.

Reply:
Probably just a small oversight while handling a merge conflict, no big deal.